kustomize/commands/localize/localize_test.go

```diff
@@ -39,6 +39,22 @@ spec:
         - containerPort: 80
 `
 
+const helmKustomization = `apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+helmCharts:
+  - name: helmTest
+    repo: testRepo
```

Comment:
If you'd like to have something that pulls real content from somewhere, we have quite a few usages of
```
- name: external-dns
  repo: oci://registry-1.docker.io/bitnamicharts/
```
in tests.

Reply:
thank you!